diff --git a/source/icudefs.mk.in b/source/icudefs.mk.in
--- a/source/icudefs.mk.in
+++ b/source/icudefs.mk.in
@@ -101,10 +101,10 @@ pkgsysconfdir = $(sysconfdir)/$(PACKAGE)$(ICULIBSUFFIX)
 
 MKINSTALLDIRS = $(SHELL) $(top_srcdir)/mkinstalldirs
 
-INSTALL = @INSTALL@
-INSTALL_PROGRAM = @INSTALL_PROGRAM@
-INSTALL_DATA = @INSTALL_DATA@
-INSTALL_SCRIPT = @INSTALL_SCRIPT@
+INSTALL = $(SHELL) $(top_srcdir)/install-sh -c
+INSTALL_PROGRAM = ${INSTALL}
+INSTALL_DATA = ${INSTALL} -m 644
+INSTALL_SCRIPT = ${INSTALL}
 
 # Library suffix (to support different C++ compilers)
 
